pkg/recipes/terraform/config/providers/azure.go

```diff
@@ -39,13 +39,20 @@ import (
 const (
 	AzureProviderName = "azurerm"
 
-	azureFeaturesParam               = "features"
-	azureSubIDParam                  = "subscription_id"
-	azureClientIDParam               = "client_id"
-	azureClientSecretParam           = "client_secret"
-	azureTenantIDParam               = "tenant_id"
-	azureUseAKSWorkloadIdentityParam = "use_aks_workload_identity"
-	azureUseCLIParam                 = "use_cli"
+	azureFeaturesParam          = "features"
+	azureSubIDParam             = "subscription_id"
+	azureClientIDParam          = "client_id"
+	azureClientSecretParam      = "client_secret"
+	azureTenantIDParam          = "tenant_id"
+	azureUseOIDCParam           = "use_oidc"
+	azureUseCLIParam            = "use_cli"
+	azureOIDCTokenFilePathParam = "oidc_token_file_path"
+
+	// The Azure AD Workload Identity Mutating Admission Webhook projects a signed service account token to
+	// this well known path.
+	// https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html
+	// https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#argument-reference
+	azureOIDCTokenFilePath = "/var/run/secrets/azure/tokens/azure-identity-token"
```

Comment:
Q: will we be able to retrieve the TokenFilePath in the fetchAzureCredentials() call (eg. line 153 below )  instead of the const value here, say, in a future PR?

Reply:
we totally could - if we have a user ask us to make this configurable then we can. for now it looks like most if not everyone uses the standard well known file path established by the azure ad workload identity project.